src/JobManager/VertexManager.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 IROHA LAB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Job } from '../entity/Job';
+import { EventEmitter } from 'events';
+
+export interface VertexManager {
```

Comment:
nit: I wonder why this needs an interface

Reply:
In order to Inject two different implementations one is the actual VertexManager, the VertexManagerImpl, and another is the FakeVertexManager for the unit test of the JobManager.spec.ts.